src/stitching/errors.ts

```diff
@@ -0,0 +1,91 @@
+import { GraphQLResolveInfo, responsePathAsArray } from 'graphql';
+import { locatedError } from 'graphql/error';
+
+const ERROR_SYMBOL = Symbol('Schema Merging Error');
```

Comment:
Maybe something like `subSchemaErrors` as the symbol name?

Reply:
Done.